extensions/github-authentication/build/postinstall.js

```diff
@@ -20,7 +20,9 @@ function main() {
 		}
 	}
 
-	fs.writeFileSync(path.join(__dirname, '../src/common/config.json'), JSON.stringify(content));
+	if (content !== {}) {
```

Comment:
Pretty sure that condition won't work. How about this?

```suggestion
	if (Object.keys(content).length > 0) {
```

Reply:
D'oh! 🤦‍♀ Thanks